commit ca6790e0208a5513098a9657420a8261af077ded
Author: QbProg <tholag@gmail.com>
Date:   Thu Mar 17 13:12:18 2011 +0100

    Replaced guards for MSVC specific pragmas from WNT to _MSC_VER
    
    These are mainly #pragma warnings or MSVCRT specific things.
    Left #pragma comment (lib, "") because other compilers seem to support these too.

Index: oce/src/BOPTools/BOPTools_PaveSet.cxx
===================================================================
--- oce.orig/src/BOPTools/BOPTools_PaveSet.cxx
+++ oce/src/BOPTools/BOPTools_PaveSet.cxx
@@ -11,7 +11,7 @@
 #include <BOPTools_QuickSortPave.hxx>
 #include <BOPTools_ComparePave.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning ( disable : 4101 )
 #endif
 
@@ -75,6 +75,6 @@
     }
   }
 }
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning ( default : 4101 )
 #endif
Index: oce/src/BinDrivers/BinDrivers.cxx
===================================================================
--- oce.orig/src/BinDrivers/BinDrivers.cxx
+++ oce/src/BinDrivers/BinDrivers.cxx
@@ -18,7 +18,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/BinLDrivers/BinLDrivers.cxx
===================================================================
--- oce.orig/src/BinLDrivers/BinLDrivers.cxx
+++ oce/src/BinLDrivers/BinLDrivers.cxx
@@ -15,7 +15,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/BinTObjDrivers/BinTObjDrivers.cxx
===================================================================
--- oce.orig/src/BinTObjDrivers/BinTObjDrivers.cxx
+++ oce/src/BinTObjDrivers/BinTObjDrivers.cxx
@@ -17,7 +17,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/BinXCAFDrivers/BinXCAFDrivers.cxx
===================================================================
--- oce.orig/src/BinXCAFDrivers/BinXCAFDrivers.cxx
+++ oce/src/BinXCAFDrivers/BinXCAFDrivers.cxx
@@ -13,7 +13,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/BooleanOperations/BooleanOperations_ShapesDataStructure.cxx
===================================================================
--- oce.orig/src/BooleanOperations/BooleanOperations_ShapesDataStructure.cxx
+++ oce/src/BooleanOperations/BooleanOperations_ShapesDataStructure.cxx
@@ -44,7 +44,7 @@
 #include <BOPTColStd_Dump.hxx>
 
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning ( disable : 4291 )
 #endif
 
Index: oce/src/DPrsStd/DPrsStd.cxx
===================================================================
--- oce.orig/src/DPrsStd/DPrsStd.cxx
+++ oce/src/DPrsStd/DPrsStd.cxx
@@ -22,7 +22,7 @@
 #include <OSD_File.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 //static Handle(AppStdL_Application) stdApp;
Index: oce/src/FWOSDriver/FWOSDriver.cxx
===================================================================
--- oce.orig/src/FWOSDriver/FWOSDriver.cxx
+++ oce/src/FWOSDriver/FWOSDriver.cxx
@@ -10,7 +10,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/IntTools/IntTools.cxx
===================================================================
--- oce.orig/src/IntTools/IntTools.cxx
+++ oce/src/IntTools/IntTools.cxx
@@ -23,7 +23,7 @@
 #include <TColStd_ListIteratorOfListOfReal.hxx>
 #include <gce_ErrorType.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning ( disable : 4101 )
 #endif
 //=======================================================================
@@ -366,6 +366,6 @@
   return 1;
 }
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning ( default : 4101 )
 #endif
Index: oce/src/IntTools/IntTools_EdgeFace.cxx
===================================================================
--- oce.orig/src/IntTools/IntTools_EdgeFace.cxx
+++ oce/src/IntTools/IntTools_EdgeFace.cxx
@@ -55,7 +55,7 @@
 #include <GeomAdaptor_HSurface.hxx>
 #include <IntCurveSurface_IntersectionPoint.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning ( disable : 4101 )
 #endif
 
@@ -1596,6 +1596,6 @@
 }
 
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning ( default : 4101 )
 #endif
Index: oce/src/LDOM/LDOM_BasicAttribute.cxx
===================================================================
--- oce.orig/src/LDOM/LDOM_BasicAttribute.cxx
+++ oce/src/LDOM/LDOM_BasicAttribute.cxx
@@ -7,7 +7,7 @@
 #include <LDOM_BasicAttribute.hxx>
 #include <LDOM_MemManager.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 // Disable the warning: "operator new unmatched by delete"
 #pragma warning (disable:4291)
 #endif
Index: oce/src/LDOM/LDOM_BasicElement.cxx
===================================================================
--- oce.orig/src/LDOM/LDOM_BasicElement.cxx
+++ oce/src/LDOM/LDOM_BasicElement.cxx
@@ -9,7 +9,7 @@
 #include <LDOM_BasicText.hxx>
 #include <LDOM_MemManager.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 // Disable the warning: "operator new unmatched by delete"
 #pragma warning (disable:4291)
 #endif
Index: oce/src/LDOM/LDOM_BasicText.cxx
===================================================================
--- oce.orig/src/LDOM/LDOM_BasicText.cxx
+++ oce/src/LDOM/LDOM_BasicText.cxx
@@ -8,7 +8,7 @@
 #include <LDOM_BasicText.hxx>
 #include <LDOM_MemManager.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 // Disable the warning: "operator new unmatched by delete"
 #pragma warning (disable:4291)
 #endif
Index: oce/src/OSD/OSD_Environment.cxx
===================================================================
--- oce.orig/src/OSD/OSD_Environment.cxx
+++ oce/src/OSD/OSD_Environment.cxx
@@ -215,7 +215,9 @@
 
 #include <windows.h>
 
+#ifdef _MSC_VER
 #pragma warning( disable : 4700 )
+#endif
 
 static void __fastcall _set_error ( OSD_Error&, DWORD );
 
Index: oce/src/OSD/OSD_Process.cxx
===================================================================
--- oce.orig/src/OSD/OSD_Process.cxx
+++ oce/src/OSD/OSD_Process.cxx
@@ -214,8 +214,9 @@
 #include <OSD_WNT_1.hxx>
 #include <LMCONS.H> /// pour UNLEN  ( see MSDN about GetUserName() )
 
-
+#ifdef _MSC_VER
 #pragma warning( disable : 4700 )
+#endif
 
 void _osd_wnt_set_error ( OSD_Error&, OSD_WhoAmI, ... );
 
Index: oce/src/OpenGl/OpenGl_GraphicDriver.cxx
===================================================================
--- oce.orig/src/OpenGl/OpenGl_GraphicDriver.cxx
+++ oce/src/OpenGl/OpenGl_GraphicDriver.cxx
@@ -31,7 +31,7 @@
 // classe OSD_SharedLibrary dans la methode SetGraphicDriver de la
 // classe Graphic3d_GraphicDevice
 extern "C" {
-#ifdef WNT /* disable MS VC++ warning on C-style function returning C++ object */
+#ifdef _MSC_VER /* disable MS VC++ warning on C-style function returning C++ object */
 #pragma warning(push)
 #pragma warning(disable:4190)
 #endif
@@ -43,7 +43,7 @@
       return aOpenDriver;
       //              return new OpenGl_GraphicDriver (AShrName);
     }
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(pop)
 #endif
 }
Index: oce/src/PS/PS_Driver.cxx
===================================================================
--- oce.orig/src/PS/PS_Driver.cxx
+++ oce/src/PS/PS_Driver.cxx
@@ -106,7 +106,7 @@
 #include <Image_Image.hxx>
 #include <stdio.h>
 static Handle(Image_Image) myImage;
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning (disable : 4244 4018 4101)
 #endif
 
Index: oce/src/PlotMgt/PlotMgt_TextManager.cxx
===================================================================
--- oce.orig/src/PlotMgt/PlotMgt_TextManager.cxx
+++ oce/src/PlotMgt/PlotMgt_TextManager.cxx
@@ -6,7 +6,7 @@
 #include <PlotMgt_TextManager.ixx>
 #include <Aspect_Units.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning (disable : 4244)
 #endif
 
Index: oce/src/Poly/Poly_CoherentNode.cxx
===================================================================
--- oce.orig/src/Poly/Poly_CoherentNode.cxx
+++ oce/src/Poly/Poly_CoherentNode.cxx
@@ -7,7 +7,7 @@
 #include <Poly_CoherentNode.hxx>
 #include <Poly_CoherentTriangle.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(disable:4291 4996)
 #endif
 
Index: oce/src/Poly/Poly_CoherentTriPtr.cxx
===================================================================
--- oce.orig/src/Poly/Poly_CoherentTriPtr.cxx
+++ oce/src/Poly/Poly_CoherentTriPtr.cxx
@@ -6,7 +6,7 @@
 
 #include <Poly_CoherentTriPtr.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(disable:4291)
 #endif
 
Index: oce/src/StdDrivers/StdDrivers.cxx
===================================================================
--- oce.orig/src/StdDrivers/StdDrivers.cxx
+++ oce/src/StdDrivers/StdDrivers.cxx
@@ -16,7 +16,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/StdLDrivers/StdLDrivers.cxx
===================================================================
--- oce.orig/src/StdLDrivers/StdLDrivers.cxx
+++ oce/src/StdLDrivers/StdLDrivers.cxx
@@ -15,7 +15,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/TDF/TDF_Data.cxx
===================================================================
--- oce.orig/src/TDF/TDF_Data.cxx
+++ oce/src/TDF/TDF_Data.cxx
@@ -75,7 +75,7 @@
   aDelta->AddAttributeDelta(DELTACREATION); \
 }
 
-#ifdef WNT
+#ifdef _MSC_VER
 // Disable the warning: "operator new unmatched by delete"
 #pragma warning (disable:4291)
 #endif
@@ -98,7 +98,7 @@
   myRoot = new (anIncAllocator) TDF_LabelNode (this);
 }
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning (default:4291)
 #endif
 
Index: oce/src/TDF/TDF_Label.cxx
===================================================================
--- oce.orig/src/TDF/TDF_Label.cxx
+++ oce/src/TDF/TDF_Label.cxx
@@ -308,7 +308,7 @@
   }
 }
 
-#ifdef WNT
+#ifdef _MSC_VER
 // Disable the warning: "operator new unmatched by delete"
 #pragma warning (disable:4291)
 #endif
@@ -373,7 +373,7 @@
   return childLabelNode;
 }
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning (default:4291)
 #endif
 
Index: oce/src/TObjDRAW/TObjDRAW.cxx
===================================================================
--- oce.orig/src/TObjDRAW/TObjDRAW.cxx
+++ oce/src/TObjDRAW/TObjDRAW.cxx
@@ -31,7 +31,7 @@
 #include <Draw_PluginMacro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/TestTopOpe/TestTopOpe.cxx
===================================================================
--- oce.orig/src/TestTopOpe/TestTopOpe.cxx
+++ oce/src/TestTopOpe/TestTopOpe.cxx
@@ -13,7 +13,7 @@
 #include <Draw_Interpretor.hxx>
 #include <Draw_PluginMacro.hxx>
 #include <BOPTest.hxx>
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/TopExp/TopExp_Explorer.cxx
===================================================================
--- oce.orig/src/TopExp/TopExp_Explorer.cxx
+++ oce/src/TopExp/TopExp_Explorer.cxx
@@ -127,7 +127,7 @@
 //purpose  : 
 //=======================================================================
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(push)
 #pragma warning(disable: 4291) // to avoid warning when using new(buffer) syntax
 #endif
@@ -209,7 +209,7 @@
   hasMore = Standard_False;
 }
 
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(pop)
 #endif
 
Index: oce/src/ViewerTest/ViewerTest.cxx
===================================================================
--- oce.orig/src/ViewerTest/ViewerTest.cxx
+++ oce/src/ViewerTest/ViewerTest.cxx
@@ -53,7 +53,7 @@
 #include <Viewer2dTest.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning(4:4190)
 #pragma warning (disable:4996)
Index: oce/src/ViewerTest/ViewerTest_ObjectCommands.cxx
===================================================================
--- oce.orig/src/ViewerTest/ViewerTest_ObjectCommands.cxx
+++ oce/src/ViewerTest/ViewerTest_ObjectCommands.cxx
@@ -78,7 +78,7 @@
 #include <strings.h>
 #endif
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/ViewerTest/ViewerTest_ViewerCommands.cxx
===================================================================
--- oce.orig/src/ViewerTest/ViewerTest_ViewerCommands.cxx
+++ oce/src/ViewerTest/ViewerTest_ViewerCommands.cxx
@@ -40,8 +40,10 @@
 #include <WNT_WClass.hxx>
 #include <WNT_Window.hxx>
 
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
+#endif
 
 #endif
 
Index: oce/src/VrmlData/VrmlData_Geometry.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_Geometry.cxx
+++ oce/src/VrmlData/VrmlData_Geometry.cxx
@@ -23,7 +23,7 @@
 #include <NCollection_Vector.hxx>
 #include <Standard_ErrorHandler.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/VrmlData/VrmlData_Group.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_Group.cxx
+++ oce/src/VrmlData/VrmlData_Group.cxx
@@ -14,7 +14,7 @@
 #include <Precision.hxx>
 #include <gp_Ax1.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/VrmlData/VrmlData_IndexedFaceSet.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_IndexedFaceSet.cxx
+++ oce/src/VrmlData/VrmlData_IndexedFaceSet.cxx
@@ -20,7 +20,7 @@
 #include <Poly.hxx>
 #include <TShort_HArray1OfShortReal.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/VrmlData/VrmlData_IndexedLineSet.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_IndexedLineSet.cxx
+++ oce/src/VrmlData/VrmlData_IndexedLineSet.cxx
@@ -15,7 +15,7 @@
 #include <TopoDS_Edge.hxx>
 #include <TopoDS_Wire.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/VrmlData/VrmlData_Material.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_Material.cxx
+++ oce/src/VrmlData/VrmlData_Material.cxx
@@ -9,7 +9,7 @@
 #include <VrmlData_Scene.hxx>
 #include <gp_XYZ.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/VrmlData/VrmlData_Node.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_Node.cxx
+++ oce/src/VrmlData/VrmlData_Node.cxx
@@ -14,7 +14,7 @@
 #include <gp_XY.hxx>
 #include <gp_XYZ.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/VrmlData/VrmlData_Scene.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_Scene.cxx
+++ oce/src/VrmlData/VrmlData_Scene.cxx
@@ -33,7 +33,7 @@
 #include <BRep_Builder.hxx>
 #include <Precision.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/VrmlData/VrmlData_WorldInfo.cxx
===================================================================
--- oce.orig/src/VrmlData/VrmlData_WorldInfo.cxx
+++ oce/src/VrmlData/VrmlData_WorldInfo.cxx
@@ -8,7 +8,7 @@
 #include <VrmlData_Scene.hxx>
 #include <VrmlData_InBuffer.hxx>
 
-#ifdef WNT
+#ifdef _MSC_VER
 #define _CRT_SECURE_NO_DEPRECATE
 #pragma warning (disable:4996)
 #endif
Index: oce/src/XCAFDrivers/XCAFDrivers.cxx
===================================================================
--- oce.orig/src/XCAFDrivers/XCAFDrivers.cxx
+++ oce/src/XCAFDrivers/XCAFDrivers.cxx
@@ -20,7 +20,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/XDEDRAW/XDEDRAW.cxx
===================================================================
--- oce.orig/src/XDEDRAW/XDEDRAW.cxx
+++ oce/src/XDEDRAW/XDEDRAW.cxx
@@ -75,7 +75,7 @@
 
 #define ZVIEW_SIZE 1000000.0
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/XSDRAWSTLVRML/XSDRAWSTLVRML.cxx
===================================================================
--- oce.orig/src/XSDRAWSTLVRML/XSDRAWSTLVRML.cxx
+++ oce/src/XSDRAWSTLVRML/XSDRAWSTLVRML.cxx
@@ -61,7 +61,7 @@
 
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/XmlDrivers/XmlDrivers.cxx
===================================================================
--- oce.orig/src/XmlDrivers/XmlDrivers.cxx
+++ oce/src/XmlDrivers/XmlDrivers.cxx
@@ -24,7 +24,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/XmlLDrivers/XmlLDrivers.cxx
===================================================================
--- oce.orig/src/XmlLDrivers/XmlLDrivers.cxx
+++ oce/src/XmlLDrivers/XmlLDrivers.cxx
@@ -19,7 +19,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/XmlTObjDrivers/XmlTObjDrivers.cxx
===================================================================
--- oce.orig/src/XmlTObjDrivers/XmlTObjDrivers.cxx
+++ oce/src/XmlTObjDrivers/XmlTObjDrivers.cxx
@@ -18,7 +18,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
Index: oce/src/XmlXCAFDrivers/XmlXCAFDrivers.cxx
===================================================================
--- oce.orig/src/XmlXCAFDrivers/XmlXCAFDrivers.cxx
+++ oce/src/XmlXCAFDrivers/XmlXCAFDrivers.cxx
@@ -14,7 +14,7 @@
 #include <Plugin_Macro.hxx>
 
 // avoid warnings on 'extern "C"' functions returning C++ classes
-#ifdef WNT
+#ifdef _MSC_VER
 #pragma warning(4:4190)
 #endif
 
